cairo-sys-rs/src/lib.rs

```diff
@@ -21,6 +21,7 @@ pub mod enums;
 
 use enums::{
     Status,
+    Content,
```

Comment:
@EPashkin: Except if you have strong feelings against this implementation, I'm ok with it.